sunpy/map/sources/psp.py

```diff
@@ -36,6 +36,10 @@ def __init__(self, data, header, **kwargs):
         self.plot_settings['norm'] = ImageNormalize(
             stretch=AsinhStretch(a=0.001), vmin=0)
 
+    @property
+    def reference_date(self):
```

Comment:
There is a unit test which was asserting this, but I don't know if that's because the test was written to test the default map behaviour or if it was encoding actual knowledge of WISPR data.

I wonder if @svank happens to know which date header in a WISPR file is the one that corresponds to the spacecraft location?

Reply:
WISPR documents the DATE-* cards on page 8 [here](https://wispr.nrl.navy.mil/sites/wispr.nrl.navy.mil/files/wispr_data_user_guide_v3.pdf). `DATE-OBS` is `DATE-BEG`, and `DATE-AVG` is my choice for a representative spacecraft location. (Each WISPR image is an on-spacecraft sum of 5-ish exposures, running from `BEG` to `END`)

Reply:
I shall swap this to use DATE-AVG on your recommendation, but it would be nice to know from the WISPR team what date stamp they use for the corresponding location they put in the headers.

Reply:
For one sample file, I can reproduce the HCI coords in the header using `DATE-AVG`, so that must be it.